packages/dx-react-grid/src/components/table-layout/virtual-table-utils.js

```diff
@@ -0,0 +1,68 @@
+const OVERSCAN = 3;
+
+const STUB_TYPE = 'stub';
+const VISIBLE_TYPE = 'visible';
+const OVERSCAN_TYPE = 'overscan';
+
+const STARTING_KEY = 'starting';
+const ENDING_KEY = 'ending';
+
+export const getVisibleRows = (rows, top, height, getRowHeight) => {
```

Comment:
The `height` arg is used only for the `bottom` calculation. Can we pass `bottom` to `getVisibleRows`? Then, we can replace `const size = getRowHeight(row);` to the `const height = getRowHeight(row);`. It's more eloquent.